### PR TITLE
feat: upgrade hooks | audit logs

### PR DESCRIPTION
**Pull request by IAmAnubhavSaini**
## Description

Upgrade hooks

### CODE REVIEW

File: app/client/src/ce/pages/Upgrade/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  & .left {
+    min-width: 362px;
+    width: 50%;
+    flex-grow: 9;
+    display: flex;
+    flex-direction: column;
+
+    //& .text-container {
+    //  width: 288px;
+    //}
```
Comment (ankitakinger) [inline app/client/src/ce/pages/Upgrade/Footer.tsx:30]: @IAmAnubhavSaini should we remove these comments?

File: app/client/src/ce/pages/Upgrade/UpgradePage.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+export const ExternalContainer = styled.div`
+  display: flex;
+  flex-grow: 1;
+  justify-content: center;
+  align-items: center;
+  min-height: calc(100vh - 50px);
+  max-height: 100vh;
+  border-left: thin solid var(--appsmith-color-black-50);
+  background-color: var(--ads-color-black-50);
+  //width: 1180px;
```
Comment (ankitakinger) [inline app/client/src/ce/pages/Upgrade/UpgradePage.tsx:17]: @IAmAnubhavSaini this one as well

**Review by ankitakinger — APPROVED** [review]

**Comment by IAmAnubhavSaini** [comment]
@ankitakinger UpgradePage, Footer, Header, and Carousel will end up in design-system.

**Comment by ankitakinger** [comment]
> @ankitakinger UpgradePage, Footer, Header, and Carousel will end up in design-system.

Shouldn't we then add those in `ads` folder instead of `ce` folder?

**Comment by IAmAnubhavSaini** [comment]
> > @ankitakinger UpgradePage, Footer, Header, and Carousel will end up in design-system.
> 
> @IAmAnubhavSaini Shouldn't we then add those in `ads` folder instead of `ce` folder?

Didn't know that. I am moving it out as soon as this is merged.

**Review by ankitakinger — APPROVED** [review]

**Comment by IAmAnubhavSaini** [comment]
Failing tests: JSObject and git import. These are not at all releated to the current changeset.

Screenshot 2022-10-17 at 3 34 03 PM

**Pull request merged**